app/src/main/res/values/dimens.xml

```diff
@@ -364,6 +364,7 @@
   <dimen name="profile_chooser_profile_view_textview_margin_start_profile_not_added">32dp</dimen>
 
   <dimen name="content_item_question_view_margin_top">36dp</dimen>
+
```

Comment:
Why this newline?

Reply:
Was going to add dimensions here but ended up not, thanks for catching this

Reply:
Done, line removed